.firebaserc

```diff
@@ -1,5 +1,5 @@
 {
   "projects": {
-    "default": "hedwig-demo"
+    "default": "hedwig-fb8a6"
```

Comment:
Please revert this. Consider that the file will be removed with #160, any edit to the file should not be committed